nvflare/private/fed/server/job_cmds.py

```diff
@@ -327,7 +327,10 @@ def list_jobs(self, conn: Connection, args: List[str]):
                 filtered_jobs.sort(key=lambda job: job.meta.get(JobMetaKey.SUBMIT_TIME.value, 0.0), reverse=reverse)
```

Comment:
Maybe default we should have "reverse" equal to True and keep the following line unchanged?

Reply:
let's say the order from old to latest: job1 job2 job3 job4,

if specified nothing we want:
job1
job2
job3
job4

if specified -m 1 we want:
job4

-m 2:
job3
job4